src/components/Home/index.js

```diff
@@ -13,6 +13,7 @@ import axios from 'axios';
 import NewProblemsForm from './components/NewProblemsForm';
```

Comment:
It's getting quite big. Is there a way to split this component to smaller ones? Maybe we could put some of part of the state into the child components? For example maybe if there were a component especially for the modals it would improve the readability